RdlMigration/ConvertRDL.cs

```diff
@@ -45,16 +51,25 @@ public void ConvertFolder(string urlEndPoint, string inputPath, string workspace
 
             if (!rdlFileIO.IsFolder(inputPath))
             {
+                rootFolder = Path.GetDirectoryName(inputPath).Replace("\\", "/");
+                var reportName = Path.GetFileName(inputPath);
+                reportNameMap.TryAdd(reportName, inputPath);
```

Comment:
need to handle failure?

Reply:
fail cases (if the report doesn't exist) are handled by the "IsFolder" function. it will throw an exception which is logged on the screen.